src/test/java/hw2/ex1/FirstExerciseTest.java

```diff
@@ -0,0 +1,104 @@
+package hw2.ex1;
+
+import hw2.common.AbstractBaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class FirstExerciseTest extends AbstractBaseTest {
+
+    @Test
+    public void firstExerciseTest() {
+
+        // 1. Open test site by URL
+        driver.get("https://epam.github.io/JDI/");
+
+        // 2. Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        // 3. Perform login
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.id("name")).sendKeys("epam");
+        driver.findElement(By.id("password")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+
+        // 4. Assert User name in the left-top side of screen that user is loggined
+        String actualLoggedUser = driver.findElement(By.id("user-name")).getText();
+        assertEquals(actualLoggedUser, "PITER CHAILOVSKII");
+
+        // 5. Assert Browser title
+        assertEquals(driver.getTitle(), "Home Page");
+
+        // 6. Assert that there are 4 items on the header section
+        // are displayed and they have proper texts
+        List<String> expectedHeaderListItems = Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
+        List<WebElement> headerListItems = driver.findElements(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8']/li"));
```

Comment:
Why do you decide specify all classes in the locator?

Reply:
Necessary elements are found only with using all classes

Reply:
Locator with contain is used now